src/coreclr/binder/bindertracing.cpp

```diff
@@ -160,6 +159,11 @@ namespace
     const WCHAR *s_assemblyNotFoundMessage = W("Could not locate assembly");
 }
 
+void GetAssemblyLoadContextNameFromBinderForExceptionHandling(AssemblyBinder* binder, /*out*/ SString& alcName)
```

Comment:
Instead of adding this bridge method, I can move the whole thing to some shared place. Opinions?

Reply:
I'd prefer moving to a shared place. Keeping it under bindertracing now that other things use it is non-intuitive to me.

Reply:
Moved to assemblybinder.cpp/.h